app/views/_controls.html.erb

```diff
@@ -9,7 +9,7 @@
               <li <%= 'class=active' if request.url.include? 'dashboard' %>><a href="/dashboard/" >DASHBOARD</a></li>
               <% end %>
               <li <%= 'class=active' if current_page?(sufia.about_path) %>><a href="/about/" >ABOUT</a></li>
-              <li <%= 'class=active' if current_page?(help_path) %>><a href="/help/" >HELP</a></li>
+              <li><a href="http://guides.lib.vt.edu/" >HELP</a></li>

```

Comment:
In our last meeting, didn't someone make the point that if we collect all in one place (like the locale file) all the terms/links/etc. that might be subject to change then it would make it easier/quicker to find and change them in the future?